nixos/modules/system/boot/systemd.nix

```diff
@@ -725,6 +725,8 @@ in
 
       "systemd/system.conf".text = ''
         [Manager]
+        DefaultCPUAccounting=yes
+        DefaultBlockIOAccounting=yes
```

Comment:
The manual can explain it better: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html#CPUAccounting=